etna/analysis/plotters.py

```diff
@@ -729,3 +733,69 @@ def plot_residuals(
         ax[i].set_title(segment)
         ax[i].tick_params("x", rotation=45)
         ax[i].set_xlabel(feature)
+
+
+TrendTransformType = Union[
+    "ChangePointsTrendTransform", "LinearTrendTransform", "TheilSenTrendTransform", "STLTransform"
+]
+
+
+def __get_labels_names(trend_transform, segments):
+    """If only unique transform classes are used then show their short names (without parameters). Otherwise show their full repr as label."""
+    from etna.transforms.decomposition.detrend import LinearTrendTransform
+    from etna.transforms.decomposition.detrend import TheilSenTrendTransform
+
+    labels = [transform.__repr__() for transform in trend_transform]
+    labels_short = [i[: i.find("(")] for i in labels]
+    if len(np.unique(labels_short)) == len(labels_short):
+        labels = labels_short
+    linear_coeffs = dict(zip(segments, ["" for i in range(len(segments))]))
+    if len(trend_transform) == 1 and isinstance(trend_transform[0], (LinearTrendTransform, TheilSenTrendTransform)):
+        for seg in segments:
+            linear_coeffs[seg] = ", k=" + str(trend_transform[0].segment_transforms[seg]._linear_model.coef_[0])
+    return labels, linear_coeffs
+
+
+def plot_trend(
+    ts: "TSDataset",
+    trend_transform: Union["TrendTransformType", List["TrendTransformType"]],
+    segments: Optional[List[str]] = None,
+    columns_num: int = 2,
+    figsize: Tuple[int, int] = (10, 5),
+):
+    """Plot series and trend from trend transform for this series.
+
+    If only unique transform classes are used then show their short names (without parameters). Otherwise show their full repr as label
+
+    Parameters
+    ----------
+    ts:
+        dataframe of timeseries that was used for trend plot
+    trend_transform:
+        trend transform or list of trend transforms to apply
+    segments:
+        segments to use
+    columns_num:
+        number of columns in subplots
+    figsize:
+        size of the figure per subplot with one segment in inches
+    """
+    if not segments:
+        segments = list(set(ts.columns.get_level_values("segment")))
+
+    ax = prepare_axes(segments=segments, columns_num=columns_num, figsize=figsize)
+    df = ts.df
+
+    if not isinstance(trend_transform, list):
+        trend_transform = [trend_transform]
+
+    df_detrend = [transform.fit_transform(df.copy()) for transform in trend_transform]
+    labels, linear_coeffs = __get_labels_names(trend_transform, segments)
```

Comment:
```suggestion
    labels, linear_coeffs = _get_labels_names(trend_transform, segments)
```
Same as above https://stackoverflow.com/a/1301369/7415703